rsearch/bin/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	search "github.com/romana/contrib/rsearch"
+	"log"
+	//	"net/http"
+	"encoding/json"
+
+	//	"io"
+)
+
+func main() {
+	var cfgFile = flag.String("c", "", "Kubernetes reverse search config file")
+	var server = flag.Bool("s", false, "Start a server")
+	var host = flag.String("h", "", "Protocol://host for client to connect to")
+	var searchTag = flag.String("r", "", "Search resources by tag")
+	flag.Parse()
+
+	done := make(chan search.Done)
+
+	config, err := search.NewConfig(*cfgFile)
+	if *host != "" {
+		config.Server.Host = *host
+	}
+
+	if err != nil {
+		fmt.Printf("Can not read config file %s, %s\n", *cfgFile, err)
```

Comment:
Only `bin/main.go` uses `fmt`. Package itself uses `log`.

But maybe i was a bit less careful with main.go since it's just using the library.